epsagon_kubernetes_subscription.sh

```diff
@@ -63,21 +63,21 @@ function test_connection {
 function send_to_epsagon {
     EPSAGON_TOKEN=$1
     ROLE_TOKEN=$2
-    CONFIG=$3
+    CONTEXT=$3
     if [ $# == 4 ]; then
-        CONTEXT=$4
-        SERVER=`kubectl config view --kubeconfig=${CONFIG} | grep -B 3 -E "[[:space:]]$CONTEXT\>" | grep -E "\<server: " | awk '{print $2}' | head -1`
-        if [ -z $SERVER ] ; then
-            echo "Could not find the server endpoint for context: ${CONTEXT}."
-            echo " Please type the server endpoint:"
-            read SERVER
-        fi
+        CONFIG=$4
+        KUBECTL="kubectl config view --kubeconfig ${CONFIG}"
```

Comment:
I guess this line and 71 should be removed?

Reply:
@maorlx 
Not really.. maybe move 71 to before the `if` and leave this one as is?
This is used in line 73

Reply:
right my confustion